src/win32/custom_action/custom_action.cc

```diff
@@ -244,21 +244,15 @@ UINT __stdcall ShutdownServer(MSIHANDLE msi_handle) {
   DEBUG_BREAK_FOR_DEBUGGER();
   std::unique_ptr<mozc::client::ClientInterface> server_client(
       mozc::client::ClientFactory::NewClient());
-  bool server_result = true;
   if (server_client->PingServer()) {
-    server_result = server_client->Shutdown();
+    if (!server_client->Shutdown()) {
+      LOG_ERROR_FOR_OMAHA();
+    }
   }
   mozc::renderer::RendererClient renderer_client;
-  const bool renderer_result = renderer_client.Shutdown(true);
-  if (!server_result) {
+  if (!renderer_client.Shutdown(true)) {
     LOG_ERROR_FOR_OMAHA();
-    return ERROR_INSTALL_FAILURE;
   }
-  if (!renderer_result) {
-    LOG_ERROR_FOR_OMAHA();
-    return ERROR_INSTALL_FAILURE;
-  }
-
   return ERROR_SUCCESS;
```

Comment:
Please add a comment about why we can return SUCCESS here.

Reply:
Done.